rotkehlchen/api/rest.py

```diff
@@ -2167,7 +2167,7 @@ def get_eth2_daily_stats(
             only_cache: bool,
     ) -> dict[str, Any]:
         try:
-            stats, filter_total_found, sum_pnl, sum_usd_value = self.rotkehlchen.chains_aggregator.get_eth2_daily_stats(  # noqa: E501
+            stats, filter_total_found, sum_pnl = self.rotkehlchen.chains_aggregator.get_eth2_daily_stats(  # noqa: E501
```

Comment:
Are we sure we want to delete this from the API? Before we used the average but I believe the USD price of ETH should already be in the globaldb. Should we change the logic and accumulate also the usd value from the inquierer? I'm not really sure if this is a metric to keep or not, personally I would want it or at least show the current value. Maybe is better to ask left also in discord